docs/source/getting_started/teams.rst

```diff
@@ -0,0 +1,132 @@
+========
+Teams
+========
+
+Teams overview
+
+.. warning::
+    Teams are not available on all Citrine Platform deployments.
+
+    Please contact support for more information.
+
+Basic Team Use
+-----------------
+
+Most commonly, the first thing you will want to do after connecting to the Citrine Platform
+is to find a certain Team. Assume that you have created a :class:`~citrine.citrine.Citrine`
+client object named ``citrine``. To list all of the Teams that you are a member of, use the
+``list`` command.
+
+.. code-block:: python
+
+    citrine.teams.list()
+
+To retrieve a Team that you are a member of, either find the Team in the list:
+
+.. code-block:: python
+
+    team_name = "Team A"
+    all_teamss = citrine.teams.list()
+    team_a = next((team for team in all_teams if team.name == team_name), None)
+
+or get the Team by name using a helper method:
+
+.. code-block:: python
+
+    from citrine.seeding.find_or_create import find_or_create_team
+    team_name = "Team A"
+    team_a = find_or_create_team(team_collection=citrine.teams, team_name=team_name)
+
+or get it by unique identifier:
+
+.. code-block:: python
+
+    team = citrine.teams.get("baaa467e-1758-43a8-97c7-76e569d0dcab")
+
+Managing Users
+--------------
+
+Admin users can manage permissions of Users in Teams.
+
+There are three types of access permissions a user can have on a team: READ, SHARE and WRITE.
```

Comment:
*repeated to myself* don't start a holy war over the Oxford comma, don't start a holy war over the Oxford comma, don't start a holy war over the Oxford comma...

=P

Reply:
I"m in favor of Oxford commas! I just forgot this time XD